src/main/java/edu/hw1/Task8.java

```diff
@@ -0,0 +1,39 @@
+package edu.hw1;
+
+public final class Task8 {
+    public static final int CELLINROWCOUNT = 8;
```

Comment:
по спецификации константы должны писаться капсом и snake_case 

```suggestion
    public final static int CELL_IN_ROW_COUNT = 8;
```